### PR TITLE
[TW-111] Settings page updates

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Updated settings descriptions to align with v8.9.1 UI and current style guide
- Reorganized content in the Working Directory section to be easier to scan
- Updated screenshots

### CODE REVIEW

File: src/pages/docs/getting-started/settings.md [hunk 1]
```diff
@@ -34,117 +34,111 @@ warning: false
...
 
-2. The [Postman console](/docs/sending-requests/troubleshooting-api-requests/#using-the-console) will display a warning for file reading errors. It will also display errors if a  collection wants to read a file outside the working directory and the setting for it is disabled.
-
-3. For Newman, it will read from the default working directory and can be modified using CLI options. File reading errors are displayed as console errors. You can utilize Newman [verbose mode](https://github.com/postmanlabs/newman#command-line-options) to find more information about these errors.
+**Be careful using files located outside your working directory.** To use files located outside your working directory when sending requests, turn on the **Allow reading files outside working directory** toggle. This option allows third-party collections to potentially read any file on your system. Use  caution, and make sure you trust all third-party collections you are using before enabling this option.
```
Comment (jkonrath-postman) [inline src/pages/docs/getting-started/settings.md:70]: Weird space between "use" and "caution" - it renders ok though.

File: src/pages/docs/getting-started/settings.md [hunk 2]
```diff
@@ -34,117 +34,111 @@ warning: false
...
-* **Automatically follow redirects:** Prevent requests that return a 300-series response from being automatically redirected.
-* **Send anonymous usage data to Postman:** Option to disable sending basic anonymous usage data (button clicks and app events) to Postman. Postman feeds usage data into product improvements.
+* **Send no-cache header:** (Recommended) Turn on this toggle to send a `Cache-Control: no-cache` header with each request. The `no-cache` directive forces the server to revalidate each request and ensures you get an up-to-date (not stale) response.
+* **Send Postman Token header:** (Recommended) Turn on this toggle to send a random Postman token in the request header. Sending a random token ensures the receiving server handles each request separately, even when the requests are sent with the same parameters. The token can also aid debugging and help you distinguish between requests on the server side.
```
Comment (jkonrath-postman) [inline src/pages/docs/getting-started/settings.md:75]: Up to you, but should we mention XhlHttpRequest in here to make sure it ends up in search?

File: src/pages/docs/getting-started/settings.md [hunk 3]
```diff
@@ -34,117 +34,111 @@ warning: false
...
 
-Import and export data in bulk inside Postman.  This will overwrite your existing collections and environments so be a little careful. It always helps to take a backup before you are importing other files. Learn more about [importing and exporting data](/docs/getting-started/importing-and-exporting-data/) in Postman.
+## Data
 
-## Add-ons
+Use the **Data** tab to perform a bulk export or import of Postman data.
 
-Download Newman, Postman's command-line companion, to integrate Postman collections with your build system or run automated tests for your API through a cron job. Learn more about [Newman](/docs/running-collections/using-newman-cli/command-line-integration-with-newman/).
+Exporting saves all of your collections and environments in a single dump file. Importing a dump file may overwrite your existing collections and environments, so use caution. It's always a good idea to make a backup before importing files. Learn more about [importing and exporting data](/docs/getting-started/importing-and-exporting-data/).
```
Comment (jkonrath-postman) [inline src/pages/docs/getting-started/settings.md:113]: This actually changed in V8 - the export doesn't *save* a dump file anymore. It brings you to a page where you request an export, and then you're emailed a link where you can later download the file.

**Review by jkonrath-postman — CHANGES_REQUESTED** [review]
A couple of super minor changes, but otherwise looks great!

File: src/pages/docs/getting-started/settings.md (hunk 1 above)
Comment (mark-anthony-postman) [reply src/pages/docs/getting-started/settings.md:70]: I removed the extra space

File: src/pages/docs/getting-started/settings.md (hunk 2 above)
Comment (mark-anthony-postman) [reply src/pages/docs/getting-started/settings.md:75]: I added back the reference to XMLHttpRequest

File: src/pages/docs/getting-started/settings.md (hunk 3 above)
Comment (mark-anthony-postman) [reply src/pages/docs/getting-started/settings.md:113]: Made it more clear that you're requesting an export that you'll download when ready

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**